Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Application/Modules/ApplicationModule.cs

```diff
@@ -9,6 +13,28 @@ public class ApplicationModule : Module
         protected override void Load(ContainerBuilder cb)
         {
             cb.Register(c => new MvxAppStart<SampleViewModel>()).As<IMvxAppStart>().SingleInstance();
+
+            // TODO [TechnicalDebt | WIP] Improve DbContext initialization
+            cb.Register(c => InitializeLocalStorageContext(c.ResolveNamed<string>(App.BootstrapParamaters.LocalAppDataFolder)));
+
+            cb.Register(c => new StringRandomizerService(c.Resolve<LocalStorageContext>())).As<IStringRandomizerService>().SingleInstance();
+        }
+
+        // TODO [TechnicalDebt | WIP] Improve DbContext initialization (this method is ugly, also change it to EnsureCreatedAsync)
+        private LocalStorageContext InitializeLocalStorageContext(string dbPath)
+        {
+
+            var filePath = Path.Combine(dbPath, "localStorage.db");
+            if (!Directory.Exists(dbPath))
+                Directory.CreateDirectory(dbPath);
+            if (!File.Exists(filePath))
+            {
+                using (var stream = File.Create(filePath)) { /* DB context creation fails on real devices if the file doesn't exit, review while improving DbContext initialization*/ }
```

Comment:
Look into the Database Initialization options for EF. Although, since you're looking for something specific to SqlLite, it may not be exactly the same.